README.md

```diff
@@ -1 +1,30 @@
-# hw01
+# HOMEWORK 1 #
+# **About Me** #
```

Comment:
Headers are automatically bolded, so don't need to include that syntax here

Reply:
You only need the hash sign at the beginning of the line
See below:

`# HOMEWORK 1 #`
# HOMEWORK 1 # 
 
` # HOMEWORK 1`
# HOMEWORK 1